Runtime/Scripts/PersistentConnection.cs

```diff
@@ -408,14 +506,16 @@ async void CompareLatencies(Statistics receivedStats)
         PropagateError(FindCloudletEventTrigger.LatencyTooHigh,
           "Latency threshold exceeded, but no other cloudlets have a meaningful improvement in latency");
       }
+      return;
     }
 
-    async Task StopEdgeEvents()
+    void StopEdgeEvents()
```

Comment:
This intended? This makes StopEdgeEvents synchronous with Close() (UI Pause if not in background thread).

Reply:
Yes this is intended.
On UI Pause we StopAllCoroutines()

<img width="1010" alt="Screen Shot 2021-06-29 at 11 09 05 AM" src="https://user-images.githubusercontent.com/24863504/123847119-0774fc00-d8cb-11eb-9455-402eefbf3095.png">
 

StopEdgeEvents () is for Closing the connection (from our convo on slack about one function for reseting variables and terminating the EdgeEvents Connection)

Reply:
Does this answer your question?

Reply:
Ok, the second context for calling it is App directed cloudlet migration.